tests/lib/rules/jsx-no-invalid-rel.js

```diff
@@ -0,0 +1,464 @@
+      output: '<a ></a>',
+      errors: [{
+        messageId: 'onlyStrings'
+      }]
+    },
+    {
+      code: '<a rel="noreferrer noopener foobar"></a>',
+      output: '<a rel="noreferrer noopener"></a>',
+      errors: [{
+        messageId: 'realRelValues'
+      }]
+    },
+    {
+      code: '<a rel={"noreferrer noopener foobar"}></a>',
+      output: '<a rel={"noreferrer noopener"}></a>',
+      errors: [{
+        messageId: 'realRelValues'
+      }]
+    },
+    {
+      code: '<a rel={"foobar noreferrer noopener"}></a>',
+      output: '<a rel={" noreferrer noopener"}></a>',
+      errors: [{
+        messageId: 'realRelValues'
+      }]
+    },
+    {
+      code: '<a rel="canonical"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<a rel="dns-prefetch"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<a rel="icon"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<a rel="manifest"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<a rel="modulepreload"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<a rel="pingback"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<a rel="preconnect"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<a rel="prefetch"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<a rel="preload"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<a rel="prerender"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<a rel="stylesheet"></a>',
+      output: '<a rel=""></a>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<area rel="canonical"></area>',
+      output: '<area rel=""></area>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<area rel="dns-prefetch"></area>',
+      output: '<area rel=""></area>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<area rel="icon"></area>',
+      output: '<area rel=""></area>',
+      errors: [{
+        messageId: 'matchingRelValues'
+      }]
+    },
+    {
+      code: '<area rel="manifest"></area>',
+      output: '<area rel=""></area>',
```

Comment:
seems like it should just remove the `rel` value entirely?

Reply:
The eslint docs state that if any fixes are applied then a subsequent linting cycle is done. I believe that that would fix this.

Reply:
While that's true, it seems ideal to avoid forcing an unnecessary fix pass, especially since in editor integrations, only one pass is run. I'd prefer to optimize this.